go.mod

```diff
@@ -3,38 +3,30 @@ module github.com/ipfs/go-graphsync
 go 1.12
 
 require (
-	github.com/btcsuite/btcd v0.0.0-20190629003639-c26ffa870fd8 // indirect
-	github.com/gogo/protobuf v1.2.1
-	github.com/golang/protobuf v1.3.2 // indirect
+	github.com/filecoin-project/go-data-transfer v0.0.0-20191219005021-4accf56bd2ce
+	github.com/gogo/protobuf v1.3.1
+	github.com/ipfs/go-bitswap v0.1.8
 	github.com/ipfs/go-block-format v0.0.2
-	github.com/ipfs/go-blockservice v0.1.0
-	github.com/ipfs/go-cid v0.0.3
-	github.com/ipfs/go-datastore v0.0.5
-	github.com/ipfs/go-ipfs-blockstore v0.0.1
+	github.com/ipfs/go-blockservice v0.1.3-0.20190908200855-f22eea50656c
```

Comment:
released v0.1.3.

Reply:
the issue here is that go-blockservice 1.3 uses go-datastore 0.4.4. Currently we are not compatible with datastore 0.4.4 in filecoin land. I think for these dependency updates I'd like to wait for the 0.5.0 IPFS release and then do them in one chunk, with the official dependencies of go-ipfs 0.5.0